insights/parsers/ps.py

```diff
@@ -130,6 +130,22 @@ def fuzzy_match(self, proc):
         """
         return any(proc in row[self.command_name] for row in self.data)
 
+    def number_occurences(self, proc):
+        """
+        Returns the number of occurencies of commands that contain given text
+
+        Returns:
+            int: The number of occurencies of commands with given text
+
+        .. note:: 'proc' can match anywhere in the command path, name or
+           arguments.
+        """
+        proc_count = 0
+        for row in self.data:
+            if proc in row[self.command_name]:
+                proc_count += 1
+        return proc_count
+
```

Comment:
@smalleni you could accomplish what you need with the existing`search` method using something like this in your own code:
```
number_of_occurences = len(ps_parser_obj.search(COMMAND_NAME__contains='text'))
```
Would this be sufficient instead of writing a new method in the parser?

Reply:
@bfahr Thanks for the review.  I guess, so. Just figured it would be nice to have this because I plan on implementing a lot of OpenStack rules and we will be relying on this extensively.

Reply:
@smalleni in that case, I'd recommend a simplification of the function to something like this:
```python
return len([True for row in self.data if proc in row[self.command_name]])
```
Please keep in mind that the data is filtered for some `PS` parsers and will not include all of the commands present on the system.  The unfiltered `PS` parsers do not include command arguments so search results may not be accurate if you are looking for command args.

Reply:
@bfahr thanks for the suggestion, amending the PR.